imap_processing/cli.py

```diff
@@ -78,19 +92,49 @@ def _parse_args():
     #     required=True,
     #     help="Full path to the file in the S3 bucket.",
     # )
+    # TODO: Remove this dependency from batch_starter, then remove it from here.
     parser.add_argument(
         "--file_path",
         type=str,
+        required=False,
+        help="Full path to the output file in the S3 bucket.",
+    )
+
+    parser.add_argument(
+        "--start-date",
+        type=str,
         required=True,
-        help="Full path to the file in the S3 bucket.",
+        help="Start time for the output data. Format: YYYY-MM-DD",
+    )
+
+    parser.add_argument(
+        "--end-date",
+        type=str,
+        required=False,
+        help="End time for the output data. If not provided, start_time will be used "
+        "for end_time. Format: YYYY-MM-DD",
+    )
+    # TODO: Will need to add some way of including pointing numbers
+
+    parser.add_argument(
+        "--version",
+        type=str,
+        required=True,
+        help="Version of the data",
     )
     parser.add_argument(
         "--dependency",
         type=str,
         required=True,
-        help=depdency_help,
+        help=dependency_help,
+    )
+
+    parser.add_argument(
+        "--use-remote",
```

Comment:
It might be better if we name this with the word "upload" somewhere in it, since that is what it is used for? Maybe `--upload-to-cloud` or `--upload-to-aws` or `--upload-to-sdc`?

Reply:
`--api-upload`?